sechub-systemtest/src/main/java/com/mercedesbenz/sechub/systemtest/config/PDSSolutionDefinition.java

```diff
@@ -0,0 +1,84 @@
+package com.mercedesbenz.sechub.systemtest.config;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class PDSSolutionDefinition extends AbstractDefinition {
+
+    private String name;
+    private String url;
+
+    private String baseDir;
+    private String pathToPdsServerConfigFile;
+
+    private CredentialsDefinition techUser = new CredentialsDefinition();
+
+    private List<ExecutionStepDefinition> start = new ArrayList<>();
+
+    private Optional<Boolean> waitForAvailable = Optional.ofNullable(null);
```

Comment:
I used optional for parts which are optional in configuration.
The defaults are only set by the system test preparation class and never 
inside the definitions.

I started with defining defaults inside the classes, but this was odd after some time, because we had no "central" part which did the default and fallbacks. 
=> Changed this and defaults are handled now all central inside preparator class.